ext/intl/breakiterator/rulebasedbreakiterator_methods.cpp

```diff
@@ -28,8 +28,9 @@ extern "C" {
 using icu::RuleBasedBreakIterator;
 using icu::Locale;
 
-static inline RuleBasedBreakIterator *fetch_rbbi(BreakIterator_object *bio) {
-	return (RuleBasedBreakIterator*)bio->biter;
+static inline RuleBasedBreakIterator *fetch_rbbi(const BreakIterator_object *bio) {
+	ZEND_ASSERT(bio != nullptr);
```

Comment:
what makes you think it ?

Reply:
You may not need this helper no more than you needed the other, especially I see in one instance (i.e. getRuleStatusVec) that the cast seems to be done needlessly twice (was already like this though) ?